templates/home/index.html.twig

```diff
@@ -1,7 +1,128 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for key, project in projects %}
+            {% for students in project.students %}
+                <li class="list-group-item">
+                    <a href="/student?name={{ students }}&amp;project={{ key }}">{{ students }}</a>
```

Comment:
Reikėtų vietoj adresų reikšmių naudoti `path` komandą.
Be to tą patį galima padaryti paprasčiau/trumpiau. Pvz.:

```twig
{{ path('student', {'name': name, 'otherParameter': other.data }) }}
```

Plačiau:
  * https://symfony.com/doc/current/templating.html#linking-to-pages
  * https://symfony.com/doc/current/reference/twig_reference.html#path